Source/WTF/wtf/threads/Signals.cpp

```diff
@@ -123,7 +123,16 @@ void initMachExceptionHandlerThread(bool enable)
 
         Config::AssertNotFrozenScope assertScope;
         SignalHandlers& handlers = g_wtfConfig.signalHandlers;
-        kern_return_t kr = mach_port_allocate(mach_task_self(), MACH_PORT_RIGHT_RECEIVE, &handlers.exceptionPort);
+
+        uint16_t flags = MPO_IMMOVABLE_RECEIVE;
```

Comment:
we don't actually want to set IMMOVABLE_RECEIVE here (and we weren't doing this previously).